controllers/admin/AdminAjaxPrestashopCheckoutController.php

```diff
@@ -461,6 +461,8 @@ public function ajaxProcessFetchOrder()
             'moduleName' => $this->module->displayName,
             'orderPayPal' => $presenter->present(),
             'orderPayPalBaseUrl' => $this->context->link->getAdminLink('AdminAjaxPrestashopCheckout'),
+            'moduleLogoUri' => $this->module->getPathUri() . 'logo.png',
+            'orderPaymentLogoUri' => $this->module->getPathUri() . 'views/img/' . strtolower($order->payment) . '.svg',
```

Comment:
Not sure if this is the right way to select payment icon

Reply:
PayPal recommends to use https://github.com/paypal/paypal-sdk-logos
But we needs to use webpack like we do for front.js
It means create a third JS app... 🤯
So I think we can use your solution to stay simple